pkg/db/db.go

```diff
@@ -21,41 +32,48 @@ type datab interface {
 	Initialize() error
 
 	PrintEnvironmentTestsAndTestCases(http.ResponseWriter, *http.Request)
+
+	PrintBasicFlake(http.ResponseWriter, *http.Request)
+
+	PrintSummary(http.ResponseWriter, *http.Request)
```

Comment:
is the Summary the overall view ? 

consider using better name than Print
if it is printing to terminal then it is Print.
if it is renderingHTML then renderOverview would a better  name

Reply:
Summary is supposed to be the overall information (so for all the environments). I don't think it's rendering or printing its writing to a json response. Is there some name to use for that?